web/src/index.css

```diff
@@ -20,4 +20,12 @@
   @apply px-4 py-2;
   @apply transition-colors hover:bg-gray-300;
   }
+
+  .h1 {
+    @apply text-2xl font-bold;
+  }
+
+  .h2 {
+      @apply text-lg font-semibold;
+    }
```

Comment:
I see this is indented. It's not a big issue but an autoformatter plugin such as `prettier` in your vs code might help here.